linkerd/app/core/src/metrics.rs

```diff
@@ -409,16 +427,25 @@ impl svc::Param<OutboundZoneLocality> for OutboundEndpointLabels {
 
 impl FmtLabels for OutboundEndpointLabels {
     fn fmt_labels(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        if let Some(a) = self.authority.as_ref() {
+        let Self {
+            server_id,
+            authority,
+            labels,
+            // TODO(kate): this label is not currently emitted.
+            zone_locality: _,
```

Comment:
in the process of writing this pull request, i found this:

this label, `zone_locality`, is not included in our outbound endpoint labels. this was added in #3262. i've refrained from adding it here, to avoid changing any behavior. should we file a ticket for this, and is this elision intentional?

Reply:
There's at least no _harm_ in omitting it, since it not at a higher cardinality than the target_addr